### PR TITLE
Updated Consul version to 1.10

### PR DESCRIPTION
**Pull request by blake**
Update Consul version to 1.10 as well as the list of supported features, and associated doc links.

### CODE REVIEW

File: _includes/comparison.html [hunk 1]
```diff
@@ -59,7 +59,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td><a href="https://istio.io/docs/tasks/traffic-management/ingress/" target="_blank">Envoy / Own Concept</a></td>
         <td>any</td>
         <td></td>
-        <td><a href="https://www.consul.io/docs/connect/gateways/ingress-gateway" target="_blank">Envoy</a> and <a href="https://www.consul.io/docs/k8s/connect/ambassador" target="_blank">Ambassador in Kubernetes</a></td>
+        <td><a href="https://www.consul.io/docs/connect/gateways/ingress-gateway" target="_blank">Envoy</a> and <a href="https://www.consul.io/docs/k8s/connect/ambassador" target="_blank">Ambassador in Kubernetes</a>, or any</td>
         <td>any</td>
```
Comment (ewolff) [inline _includes/comparison.html:63]: I am not sure I understand. The first link seems to explain that any Ingress Gateway can be managed by Consul using the Envoy proxy. The second link shows how to use the Ambassador Ingress Gateway with Consul. So maybe it should say "any with the Envoy proxy, e.g. Ambassador" and give the links for Envoy and Ambassador?

File: _includes/comparison.html [hunk 2]
```diff
@@ -200,7 +200,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td>Kubernetes</td>
         <td>Kubernetes</td>
         <td>ECS, Fargate, EKS, EC2</td>
-        <td>Kubernetes, Nomad, VMs (Universal)</td>
+        <td>ECS, Kubernetes, Nomad, VMs (Universal)</td>
```
Comment (ewolff) [inline _includes/comparison.html:203]: This seems to be a tech preview, see https://www.consul.io/docs/ecs . Should there be a note concerning its state?

File: _includes/comparison.html [hunk 3]
```diff
@@ -230,7 +230,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td><a href="https://istio.io/docs/setup/install/multicluster/" target="_blank">yes</a></td>
         <td><a href="https://linkerd.io/2/features/multicluster/" target="_blank">yes</a></td>
         <td></td>
-        <td><a href="https://www.consul.io/docs/connect/gateways/mesh-gateway" target="_blank">yes</a></td>
+        <td><a href="https://www.consul.io/docs/k8s/installation/multi-cluster/kubernetes" target="_blank">yes</a></td>
```
Comment (ewolff) [inline _includes/comparison.html:233]: Would https://www.consul.io/docs/k8s/installation/multi-cluster be the better link?

**Review by ewolff — CHANGES_REQUESTED** [review]
Thanks a lot for the PR! I have added a few comments. Please have a look. Thanks!

File: _includes/comparison.html (hunk 1 above)
Comment (blake) [reply _includes/comparison.html:63]: Its possible to use any ingress when using transparent proxy. An Envoy sidecar will be injected alongside the ingress to facilitate communication with services in the mesh.

Consul also provides its own out-of-the-box ingress gateway built on Envoy. Ambassador and Traefik (in the next release) also support native integrations with Consul which do not require an additional sidecar to be deployed.

I'll revert the change and will look to address it in a subsequent PR.

File: _includes/comparison.html (hunk 2 above)
Comment (blake) [reply _includes/comparison.html]: I added a callout that this is in tech preview.

File: _includes/comparison.html (hunk 3 above)
Comment (blake) [reply _includes/comparison.html]: That works. I updated the URL to point to the multi-cluster overview page.

**Comment by ewolff** [comment]
Great, thanks a  lot!

**Comment by blake** [comment]
@ewolff Regarding your question in https://github.com/servicemesh-es/servicemesh.es/commit/e04e3a6b62f2770e217db375a96abeba19fc92d3, I removed the link because the parameter linked here allows enabling/disabling mTLS verification for a Consul agent's RPC or HTTP endpoints. This setting does not control whether mTLS is enforced within the mesh.

Consul actually enables mTLS by default _and_ enforces it for all service-to-service connections. It is not currently possible for a user to disable TLS encryption and mTLS validation for service mesh traffic.

I hope this clarifies why the link was removed. Let me know if you have any further questions.

**Comment by ewolff** [comment]
@blake Excellent, thanks  a lot for clarifying! I will remove the link.

**Pull request merged**